html/lesson2/example.html

```diff
@@ -45,10 +45,10 @@ <h3>Owls:</h3>
         </div>
         <ul id="links">
         <li>
-           <a href="mailto:social@codebar.io?subject=I love owls :: codebar">Email us</a>
+           <a href="mailto:social@codebar.io?subject=I love owls :: Codebar">Email us</a>
```

Comment:
sorry to be pedantic but codebar is always a lowercase c, so this is deliberate :)

Reply:
Ah, no worries - there was an uppercase C here:
http://tutorials.codebar.io/html/lesson2/tutorial.html

The text
> Download the files required to begin working through the tutorial from here (mac/linux)

Links to a zip file here:
`https://gist.github.com/hundred/7332441/download`

I didn't know which was correct. 👍

Reply:
(I'll update the PR tomorrow.)

Reply:
thanks :)